classes/blocks/class-columns.php

```diff
@@ -185,6 +185,11 @@ public function prepare_data( $attributes ): array {
 			}
 		}
 
+		// enqueue script that equalizes the heights of the titles of the blocks
+		if ( ! $this->is_rest_request() ) {
+			wp_enqueue_script( 'submenu', P4GBKS_PLUGIN_URL . 'public/js/columns.js', [ 'jquery' ], '0.1', true );
```

Comment:
Hey @Inwerpsel I guess this "submenu" should be "column-headers" or something like that.

Reply:
Good catch!